OmgvaPOS/src/ItemManagement/Controller/ItemController.cs

```diff
@@ -161,5 +163,57 @@ public IActionResult DeleteItem(long id) {
             }
         }
+
+        [HttpGet("{id}/taxes")]
+        [Authorize(Roles = "Admin, Owner, Employee")]
+        [ProducesResponseType<List<TaxDto>>(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)] //TODO: Should be thrown if item does not exist.
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public IActionResult GetItemTaxes(long id) {
+            if (!JwtTokenHandler.CanManageBusiness(HttpContext.Request.Headers.Authorization!, _itemService.GetItemNoException(id).BusinessId))
+                return Forbid();
+
+            try {
+                List<TaxDto> itemTaxes = _itemService.GetItemTaxes(id);
+                return Ok(itemTaxes);
+            }
+            catch (Exception ex) {
+                _logger.LogError(ex, "An unexpected internal server error occured while retrieving item taxes.");
+                return StatusCode((int)HttpStatusCode.InternalServerError, "Internal server error.");
+            }
+        }
+
+        [HttpPost("{id}/taxes")]
+        [Authorize(Roles = "Admin, Owner, Employee")]
+        [ProducesResponseType<ItemDTO>(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public IActionResult ChangeItemTaxes([FromBody] ChangeItemTaxesRequest changeItemTaxesRequest, long id) {
+            if (!JwtTokenHandler.CanManageBusiness(HttpContext.Request.Headers.Authorization!, _itemService.GetItemNoException(id).BusinessId)) //TODO: Fix crash when inexistant business id is passed
+                return Forbid();
+
+            var allTaxIds = _taxService.GetAllTaxes().Select(t => t.Id);
+            if (!(changeItemTaxesRequest.TaxesToRemoveIds.All(taxId => allTaxIds.Contains(taxId)) 
+                && changeItemTaxesRequest.TaxesToAddIds.All(taxId => allTaxIds.Contains(taxId)))) {
+                return NotFound("Not all specified taxes exist");
+            }
+
+            try {
+                var returnItem = _itemService.ChangeItemTaxes(changeItemTaxesRequest, id);
+                if (returnItem != null) //TODO: Handle errors, handle possible null
+                    return Ok(returnItem);
+                else
+                    return NotFound();
+            }
+            catch (Exception ex) {
+                _logger.LogError(ex, "An unexpected internal server error occured while changing item taxes.");
+                return StatusCode((int)HttpStatusCode.InternalServerError, "Internal server error.");
+            }
+        }
```

Comment:
Same as above, we no longer need to explicitly handle unexpected exceptions, they will be caught and logged by our ExceptionHandler middleware